src/components/EligibilityForm/EligibilityFormAddress.tsx

```diff
@@ -25,56 +27,79 @@ const energyInputsDefaultLabels = {
 };
 
 const AddressTestForm: React.FC<CheckEligibilityFormProps> = ({
+  fullAddress,
   formLabel,
+  heatingLabel,
   energyInputsLabels = energyInputsDefaultLabels,
   centredForm,
   children,
   onChange,
   onFetch,
   onSuccess,
 }) => {
-  const [status, setStatus] = useState('idle');
-  const [data, setData] = useState({});
+  const address = fullAddress?.address;
+  const {
+    lon,
+    lat,
+    isEligible: eligibility,
+    network,
+    inZDP,
+  } = fullAddress?.addressDetails?.networkDetails || {};
+  const coords = (lon ?? lat) && { lon, lat };
+  const defaultData = {
+    address,
+    eligibility,
+    coords,
+    network,
+    inZDP,
+  };
+  const defaultStatus = !coords ? 'idle' : 'success';
```

Comment:
Why do you create a const (recreated every render, but used only on initial state)